### PR TITLE
Hod head module

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Literally what it says on the tin.
Sprite by me.

## Why It's Good For The Game
People wanted it, I suppose.

## Changelog
:cl:
add: Hod Head module has been added as a counterpart of the Hod chestpiece module
/:cl:

### CODE REVIEW

**Comment by QualityVan** [comment]
Should put it down with the hod body instead of lumped inside tyr.

**Comment by Losenis** [comment]
Speedmerge

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 1]
```diff
@@ -121,6 +121,15 @@
 	slot = ATTACHMENT_SLOT_HEAD_MODULE
 	variants_by_parent_type = list(/obj/item/clothing/head/modular/marine/m10x = "tyr_head_xn", /obj/item/clothing/head/modular/marine/m10x/leader = "tyr_head_xn")
 
+/obj/item/armor_module/module/hod_head
+	name = "Hod Helmet System"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:125]:
```suggestion
	name = "\improper Hod Helmet System"
```

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 2]
```diff
@@ -121,6 +121,15 @@
...
+	desc = "Designed for mounting on a modular helmet. When attached, this system provides substantial resistance to most gunshot wounds by providng high internal padding within the helmets structure."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:126]:
```suggestion
	desc = "Designed for mounting on a modular helmet. When attached, this system provides substantial resistance to most gunshot wounds by providing high internal padding within the helmet's structure."
```

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 3]
```diff
@@ -121,6 +121,15 @@
...
+	icon = 'icons/mob/modular/modular_armor_modules.dmi'
+	icon_state = "mod_ff"
```
Comment (Lumipharon) [inline code/modules/clothing/modular_armor/attachments/modules.dm:128]: Make a new item sprite. mimir hat is basically just a cropped mimir suit, so you can do the same.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Also yeah, move it under the main hod attachment

**Comment by Al-1ce** [comment]
> Show sprite aaaaugh

Click the thing that says 'checks' and look for IconDiff.

**Comment by bowlofcereal** [comment]
the fact that we need this speaks volumes of marine skill issue

**Review by TiviPlus — APPROVED** [review]

**Comment by Lumipharon** [comment]
whoops, meant to merge this.

**Comment by Losenis** [comment]
LETS FUCKING GOOOO

**Comment by QualityVan** [comment]
Still haven't actually moved the helmet code to be down with the chest module.

**Pull request merged**